### PR TITLE
[Docs] Japanese translation of docs/feature_terminal.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/feature_terminal.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_terminal.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    TERMINAL_ENABLE = yes
+
+そして、オンまたはオフにするために、`TERM_ON` および `TERM_OFF` キーコードを使います。
+
+有効な場合、`> ` プロンプトが現れ、入力、バックスペース (先頭に達し、オーディオが有効な場合はベルが鳴ります)、コマンドを送信するためのエンターを入力することができます。矢印キーは現在のところ無効なため、混乱することはありません。マウスでカーソルを移動することはお勧めしません。
+
+`#define TERMINAL_HELP` は、このページでは実際には必要のない他の出力ヘルパーを有効にします。
+
+"up" および "down" により、過去に入力した5つのコマンドを循環することができます。
```
Comment (shelaf) [inline docs/ja/feature_terminal.md:22]:
```suggestion
"上矢印" および "下矢印" により、過去に入力した5つのコマンドを順に切り替えることができます。
```

File: docs/ja/feature_terminal.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+ターミナル機能はテキストエディタを介してキーストロークで通信するように設計されたコマンドラインのようなインタフェースです。エディタで自動インデント機能をオフにすることは有益です。
+
+有効にするには、以下を `rules.mk` または `Makefile` に貼り付けます:
+
+    TERMINAL_ENABLE = yes
+
+そして、オンまたはオフにするために、`TERM_ON` および `TERM_OFF` キーコードを使います。
+
+有効な場合、`> ` プロンプトが現れ、入力、バックスペース (先頭に達し、オーディオが有効な場合はベルが鳴ります)、コマンドを送信するためのエンターを入力することができます。矢印キーは現在のところ無効なため、混乱することはありません。マウスでカーソルを移動することはお勧めしません。
```
Comment (shelaf) [inline docs/ja/feature_terminal.md:18]:
```suggestion
有効な場合、`> ` プロンプトが現れ、ここで文字やバックスペース (オーディオが有効な場合は、先頭に到達するとベルが鳴ります)を入力することができ、エンターを入力するとコマンドを送信します。矢印キーは現在のところ無効なため、混乱することはありません。マウスでカーソルを移動することはお勧めしません。
```

File: docs/ja/feature_terminal.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+"up" および "down" により、過去に入力した5つのコマンドを循環することができます。
+
+## 今後のアイデア
+
+* キーボード/ユーザ拡張可能なコマンド
+* より小さなフットプリント
+* 矢印キーのサポート
+* コマンド履歴 - 完了
+* SD カード サポート
```
Comment (shelaf) [inline docs/ja/feature_terminal.md:30]:
```suggestion
* SD カードのサポート
```

File: docs/ja/feature_terminal.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+* バッファディスプレイのための LCD サポート
+* Keycode -> name string LUT
+* レイヤー状態
+* *アナログ/デジタル ポートの読み込み/書き込み*
+* RGB mode stuff
```
Comment (shelaf) [inline docs/ja/feature_terminal.md:35]:
```suggestion
* RGB モード関連機能
```

File: docs/ja/feature_terminal.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+## 今後のアイデア
+
+* キーボード/ユーザ拡張可能なコマンド
+* より小さなフットプリント
+* 矢印キーのサポート
+* コマンド履歴 - 完了
+* SD カード サポート
+* バッファディスプレイのための LCD サポート
+* Keycode -> name string LUT
```
Comment (shelaf) [inline docs/ja/feature_terminal.md:32]: 自信はないですがLUT=Look Up Tableとしました。
```suggestion
* キーコード -> 名称の対応表
```

File: docs/ja/feature_terminal.md (hunk 2 above)
Comment (umi-umi) [reply docs/ja/feature_terminal.md]: 訳例ありがとうございます。自分で読み直してもすっきりしない箇所でした。。訳例をそのまま頂きます！

File: docs/ja/feature_terminal.md (hunk 4 above)
Comment (umi-umi) [reply docs/ja/feature_terminal.md]: ありがとうございます。stuffをどう訳したものやらと後回しにしていたのをすっかり忘れてしまっていました。

**Comment by mtei** [comment]
I think this is ready to merge.

**Pull request merged**